package.json

```
@@ -35,6 +35,13 @@
         "permissionName": "module.tenant-settings.enabled",
         "displayName": "UI: Tenant-settings module is enabled"
       },
+      {
+        "permissionName": "ui-tenant-setting.module.enabled",
```

It does not mean it is correct but it clearly does not correct without this change.
```suggestion
        "permissionName": "ui-tenant-settings.module.enabled",
```